make/go/dep_protoc_gen_go.mk

```diff
@@ -7,8 +7,10 @@ $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://github.com/protocolbuffers/protobuf-go/releases 20240906 checked 20240916
-PROTOC_GEN_GO_VERSION ?= v1.34.3-0.20240906163944-03df6c145d96
```

Comment:
Commit `03df6c1` is the latest commit on `master`, which includes commit `94ecbc2` from Aug 16, 2024.

I presume we don't want to be this far ahead? No strong opinion either way, we just need this commit to be at or ahead of whatever is in bufbuild/buf.